README.md

```diff
@@ -140,6 +140,11 @@ The Azure MCP Server supercharges your agents with Azure context. Here are some
 - JSON output formatting
 - Cross-platform compatibility
 
+### 📦 Azure Load Testing
+- List load tests
```

Comment:
Maybe replace these 3 bullets to match how Key Vault wrote their section (for consistency)

"List, create, and get load tests"